### PR TITLE
Counterfactual deployment

### PR DESCRIPTION
**Pull request by germartinez**
Add counterfactual deployment

### CODE REVIEW

File: packages/account-abstraction/package.json [hunk 1]
```diff
@@ -39,7 +39,9 @@
   },
   "dependencies": {
     "@gnosis.pm/safe-contracts": "^1.3.0",
+    "@gnosis.pm/safe-deployments": "^1.19.0",
```
Comment (dasanra) [inline packages/account-abstraction/package.json:42]: Remember to update to latest version

Comment (dasanra) [reply packages/account-abstraction/package.json:42]: This package is also duplicated in `devDependencies`

`prettier` and `typescript` are already available at the root `package.json` so they should not be needed here

Comment (germartinez) [reply packages/account-abstraction/package.json]: Thanks @dasanra. Changes were applied

**Review by DaniSomoza — APPROVED** [review]
Awesome job!

tested in the demo UI and works fine! :3

Captura de pantalla 2023-02-27 a las 10 55 27
Captura de pantalla 2023-02-27 a las 10 59 53
Captura de pantalla 2023-02-27 a las 11 01 09

**Pull request merged**